docs/source/start.rst

```diff
@@ -8,8 +8,12 @@ start using it.
 
 	asreview lab
 
-When you are using Windows, open `CMD.exe` and run the command. When you use
-MacOS or Linux, you can open `Terminal` and run the command.
```

Comment:
Some users report issues here. They don't open the command line, but open Python. It might be worth integrating this part of the text with lines 4-5 (and keeping the new changes as well).

Reply:
I added more details and a hyperlink to our online and free [teaching materials ](https://asreview.github.io/asreview-academy/ASReviewLAB.html)which includes screenshots.